README.md

```diff
@@ -334,18 +317,18 @@ It is possible to develop ORY Kratos on Windows, but please be aware that all gu
 
 ##### Install Tools
 
-When cloning ORY Kratos, run `make tools`. It will download several required dependencies. If you haven't run the command
+When cloning ORY Kratos, run <code textrun="is-make-task">make install</code>. It will download several required dependencies. If you haven't run the command
```

Comment:
This action checks that the Makefile contains the given task. Source code for this action is below in `text-run/is-make-task.js`.